pkg/image/api/types.go

```diff
@@ -60,6 +77,30 @@ type ImageLayer struct {
 	Size int64
 }
 
+// SignatureValidationResult holds a result of signature validation.
+type SignatureValidationResult string
+
+// ImageSignature holds a signature of an image.
+type ImageSignature struct {
+	// Describes a type of stored blob.
+	Type string
+	// An opaque binary string which is an image's signature.
+	Content []byte
+	// A time of last server-side validation. IOW a time of the last update to ValidationResult.
+	LastValidation *unversioned.Time
```

Comment:
> I wanted to emphasize that this field is tightly related to ValidationResult. If renamed to Checked, then the second field should be renamed to CheckResult which seems ambiguous to me.

Thing is, the signature hasn't been validated with respect to the image its attached to.  It's been parsed and the issuer was probably checked, but the using validation here could pretty easily be confused.

`Checked` and `SigningCertState`?